tests/load_tests/README.md

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
All this docs; now should go to docs/LoadTests.md explaining what is doing the load tests and how we can run locally/CI